web/json_value.go

```diff
@@ -169,7 +170,16 @@ func convertJSONAttributeListValue[Element any](attribute *framework.AttributeCo
 // ParseDateTime parses a timestamp against a set of predefined formats.
 func ParseDateTime(dateTimeFormats []DateTimeFormatWithTimeZone, localTimeZoneOffset int, dateTimeStr string) (dateTime time.Time, err error) {
 	for _, format := range dateTimeFormats {
-		dateTime, err = time.Parse(format.Format, dateTimeStr)
+		if format.Format == "sgnl-unix-ns-timestamp" {
+			var unixTimestamp int64
+			unixTimestamp, err = strconv.ParseInt(dateTimeStr, 10, 64)
+			if err == nil {
+				dateTime = time.Unix(unixTimestamp, 0)
```

Comment:
nit: Variable name suggesting the kind of unix timestamp
```suggestion
			var unixNanosec int64
			unixNanosec, err = strconv.ParseInt(dateTimeStr, 10, 64)
			if err == nil {
				dateTime = time.Unix(unixNanosec, 0)
```

Reply:
In the conversions, I'm making the assumption that the value provided is seconds and not nanoseconds. Time.UNIX(seconds, nanoseconds), but I'm ignoring the nanoseconds field since it is rarely used in practice. Do you still want to rename everything to this?

Reply:
In that case, let's retain this as is. 
Also, given that there's a `time.UnixDate`, let's rename the format above to `SGNLUnixTime`. WDYT?

Reply:
Ok updated to SGNLUnixSec